modules/dynamic-plugins/con-dynamic-plugins-cache.adoc

```diff
@@ -49,11 +55,75 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+[NOTE]
+To avoid adding a new volume, you must use the `$patch: replace` directive.
+
+=== Creating a PVC for Dynamic Plugins Cache using the Helm Chart
+For Helm Chart installations, if you require the dynamic plugin cache to persist across pod restarts, you must create a PersistentVolumeClaim (PVC) and configure the Helm Chart to use it.
+
+.Example creating the persistent volume definition
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-root
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+
+----
+
 [NOTE]
 ====
-Future versions of the {product-very-short} operator are planned to automatically create the PVC.
+This example uses `ReadWriteOnce` as the access mode which prevents multiple replicas from sharing the PVC across different nodes. 
+
+To run multiple replicas on different nodes, depending on your storage driver, you must use an access mode such as `ReadWriteMany`.
 ====
 
+To apply this PVC to your cluster, run the following command:
+[source,terminal]
+----
+oc apply -f pvc.yaml
+----
+
+.Example configuring Helm to use the PVC
+[source,yaml]
+----
+extraVolumes:
+  - name: dynamic-plugins-root
+    persistentVolumeClaim:
+      claimName: dynamic-plugins-root
+  - name: dynamic-plugins
+    configMap:
+      defaultMode: 420
+      name: '{{ printf "%s-dynamic-plugins" .Release.Name }}'
+      optional: true
+  - name: dynamic-plugins-npmrc
+    secret:
+      defaultMode: 420
+      optional: true
+      secretName: '{{ printf "%s-dynamic-plugins-npmrc" .Release.Name }}'
+  - name: dynamic-plugins-registry-auth
+    secret:
+      defaultMode: 416
+      optional: true
+      secretName: '{{ printf "%s-dynamic-plugins-registry-auth" .Release.Name }}'
+  - name: npmcacache
+    emptyDir: {}
+  - name: temp
+    emptyDir: {}=
+----
```

Comment:
I think it is also worth mentioning in a note that the user must repeat all the other extra volumes from the default chart in order to edit the `dynamic-plugins-root` one: https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/values.yaml#L145-L181